src/click/_bashcomplete.py

```diff
@@ -26,7 +26,7 @@
     return 0
 }
 
-%(complete_func)setup() {
+%(complete_func)ssetup() {
```

Comment:
_setup would match the zsh script

Reply:
I'm not sure I follow, are you looking for me to change this to `%(complete_func)s_setup`?

Reply:
Yes